### ISSUE TITLE
make install puts non-binaries in bin

### ISSUE THREAD

**Issue by d235j** [OSVR/OSVR-Core#238]
As currently structured, the CMakeLists put configuration files in bin. This is nonstandard.
We should instead have a share/osvr or share/osvr-core directory, and the sample configuration files, examples, and such go in there. (The docs should go in share/doc/osvr or share/doc/osvrcore, and this seems to currently be the case).

The default server config file should probably also go in the share folder.
Obviously, on Windows things might be different.

**Comment by rpavlik** [comment]
Yep - this is a remnant revealing that Windows was the first "shipping" target platform (even though we built on Linux all along for portability and robustness testing) - see also #185, #171, and #172 among others.
